net/Socket.cpp

```diff
@@ -1302,6 +1383,33 @@ LocalServerSocket::~LocalServerSocket()
 #  define LOG_CHUNK(X)
 #endif
 
+#endif // !MOBILEAPP
+
+std::string StreamSocket::toString(WSState t) noexcept
```

Comment:
Would love this non-controversial useful fluff to be in a separate commit to behavioural change. It's really good to improve our state dumping though :-)

Reply:
all commits separated and bisectable now